gulpfile.js

```diff
@@ -3,13 +3,15 @@
 var _ = require('lodash');
 var argv = require('yargs').argv;
 var gulp = require('gulp');
-var argv = require('yargs').argv;
 var gutil = require('gulp-util');
 var connect = require('gulp-connect');
-var webpack = require('webpack-stream');
+var path = require('path');
+var webpack = require('webpack');
+var webpackStream = require('webpack-stream');
```

Comment:
It's under [the Usage heading](https://github.com/shama/webpack-stream#usage).

> If you would like to use a different version of webpack than the one this plugin uses, pass in an optional 2nd argument

They're using [webpack 1.12.9](https://github.com/shama/webpack-stream/blob/master/package.json#L35).